modules/stanford_icon_grid/stanford_icon_grid.info

```diff
@@ -0,0 +1,28 @@
+name = Stanford Icon Grid Block
+core = 7.x
+package = Stanford
+version = 7.x-1.0-beta1
+dependencies[] = bean_admin_ui
+dependencies[] = ctools
+dependencies[] = features
+dependencies[] = field_collection
+dependencies[] = image
+dependencies[] = link
+dependencies[] = list
+dependencies[] = text
+explanation = Field type(s) in use - see <a href="/admin/reports/fields">Field list</a>
```

Comment:
When disabling/uninstalling the module that defines a field type with that field type in use, it will say it can't be disabled and give the explanation.